app/services/api.js

```diff
@@ -0,0 +1,19 @@
+import toPairs from 'lodash/toPairs';
+
+
+function constructQueryString(params) {
```

Comment:
Something like [query-string](https://github.com/sindresorhus/query-string) could be used here, but our use cases might be simple enough for it not to be needed.